chain/network/src/peer_manager/peer_store.rs

```diff
@@ -1,4 +1,4 @@
-use borsh::BorshSerialize;
+use borsh::{BorshDeserialize, BorshSerialize};
```

Comment:
do you need this ?

Reply:
@mm-near Yes, need it for `KnownPeerState::try_from_slice`.

By using `borsh` directly, we can avoid extra memory allocations; creating a temporary vector just to read from it.